04_numeronym/sahidakhtar1/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, len(vals))
+	for i, v := range vals {
+		result[i] = createNumeronym(v)
+	}
+	return result
+}
+
+func createNumeronym(s string) string {
+	trimedString := strings.TrimSpace(s)
+	var numeronym string
+	//If the string contains space or \t or \n then it will return the same string
+	if strings.Contains(trimedString, " ") ||
```

Comment:
https://golang.org/pkg/strings/#ContainsAny is probably the cleaner choice.

It has been specified clearly so I guess this fine, but I think the nicer solution is to work with the `Trim`ed string and treat internal space as any other character.

Reply:
Done.